.travis.yml

```diff
@@ -1,9 +0,0 @@
-dist: trusty
-language: node_js
-node_js:
-  - "12"
-  - "13"
-  - "14"
-install:
-  - npm install
-script: npm run lint && npm run build && npm test

```

Comment:
I'm just curious. Did we use travis because we weren't using github actions?

As in, are we using travis just because it was what was done before?

Reply:
> Did we use travis because we weren't using github actions?

Maybe. I don't think anyone can answer that question now, so we'll have to assume yes.

Reply:
We use TravisCI since 2017, and yes it was our first ci platform.
Currently, both GithHb actions and Travis CI work as a backup CI option for each other.

Travis CI is free for open-source projects.